src/initializers/redis.ts

```diff
@@ -1,4 +1,4 @@
-import { createClient, RedisClient } from 'redis';
+import { createClient as createClientType, RedisClient as RedisClientType } from 'redis';
```

Comment:
Isn't that still importing the module?

Reply:
No. Typescript is clever enough to not include files or code only used for typings.